wavesrv/pkg/sstore/dbops.go

```diff
@@ -854,6 +855,26 @@ func GetCmdByScreenId(ctx context.Context, screenId string, lineId string) (*Cmd
 	})
 }
 
+func UpdateWithClearOpenAICmdInfo(screenId string) (*ModelUpdate, error) {
```

Comment:
adding a lot of new functions here. I tried to make the names as descriptive as possible, let me know if they seem unclear